packages/shared/src/hooks/useDisableFilterAlert.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import request from 'graphql-request';
+import { UPDATE_ALERTS } from '../graphql/alerts';
+import { apiUrl } from '../lib/config';
+import { FeedSettings } from '../graphql/feedSettings';
+import useAlertContext from './useAlertContext';
+
+export default function useDisableFilterAlert(
+  feedSettings: FeedSettings,
+): void {
+  const { alerts, setAlerts } = useAlertContext();
+
+  useEffect(() => {
+    if (
+      alerts?.filter &&
+      (feedSettings?.includeTags?.length ||
+        feedSettings?.blockedTags?.length ||
+        feedSettings?.excludeSources?.length ||
+        feedSettings?.advancedSettings?.length)
+    ) {
+      request(`${apiUrl}/graphql`, UPDATE_ALERTS, {
+        data: { filter: false },
+      }).then(() => setAlerts({ ...alerts, filter: false }));
```

Comment:
This is not the right way to do it.
We can use an optimistic update for it. There are lots of examples throughout the codebase.
Optimistic update will update the UI even before the request is completed.